src/PerformanceTests/Common/Scenarios/SendLocalOneOnOne/SendLocalOneOnOneRunner.cs

```diff
@@ -16,9 +16,7 @@ partial class SendLocalOneOnOneRunner : BaseRunner
 
     protected override void Start()
```

Comment:
See and that's the root problem of the previous comments. When you have a
code base which moxed void and task returning APIs you will run into issues
I found above. And async call inside a void method is treated as
fire&forget. The best practice is async all the way and that for a good
reason. About your cleannes and microptimization: it is not about
microoptimization. My point is entirely about cleanness. Because currently
it is not clean in terms of async code flow

On Wednesday, April 20, 2016, Ramon Smits <notifications@github.com> wrote:

> In
> src/PerformanceTests/Common/Scenarios/SendLocalOneOnOne/SendLocalOneOnOneRunner.cs
> <https://github.com/Particular/EndToEnd/pull/61#discussion_r60491054>:
>
> > @@ -16,9 +16,7 @@ partial class SendLocalOneOnOneRunner : BaseRunner
> >
> >      protected override void Start()
>
> Because the code currently is pretty clean and adding async ceremony for
> micro-optimization is not needed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/EndToEnd/pull/61/files/abd2cd5b5e4122dd62555d09603fd491f3d16e91#r60491054>
>